### PR TITLE
Add community flag for features

### LINKED ISSUE #1062: Create a Community flag
### Discussed in https://github.com/micronaut-projects/micronaut-starter/discussions/983

<div type='discussions-op-text'>

<sup>Originally posted by **sdelamo** November 25, 2021</sup>
I would like to include a flag in features to flag them as `community`.  E.g Camunda related features or [Micronaut Agorapulse libraries](https://github.com/agorapulse/micronaut-aws-sdk).  Similar to the `preview` flag.

thoughts? 
</div>

### PR DESCRIPTION
**Pull request by timyates**
We want to be able to show if a feature is community led.

For this we have added a new community boolean to the Feature
class, and set this to true for the Camunda maintained features.

### CODE REVIEW

**Comment by timyates** [comment]
Once accepted, we need to submit a PR to `micronaut-starter-ui` to expose this also on the UI

**Review by sdelamo — COMMENTED** [review]
Are you going to change this to be an enum instead?

**Comment by timyates** [comment]
Yes I will, I'll put this back to draft for now

**Comment by timyates** [comment]
@sdelamo switched it to an Enum, but cannot use the enum as a return type, as Swagger refused to annotate it in a useful way.  I switched to returning a String from `FeatureDTO`...  Any ideas where I was going wrong?

**Comment by timyates** [comment]
@sdelamo Ready for review 👍

**Comment by timyates** [comment]
@sdelamo we've gone back to using a boolean after a discussion in gitter with @jameskleeh

**Review by sdelamo — APPROVED** [review]

File: starter-cli/src/main/java/io/micronaut/starter/cli/command/ListFeatures.java [hunk 1]
```diff
@@ -94,6 +94,9 @@ private void listFeatures(ConsoleOutput consoleOutput, Set<Feature> defaultFeatu
             if (feature.isPreview()) {
                 name += " [PREVIEW]";
             }
+            if (feature.isCommunity()) {
```
Comment (sdelamo) [inline starter-cli/src/main/java/io/micronaut/starter/cli/command/ListFeatures.java:97]: Can you add a test for this? A test which verifies the name is suffixed?

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: starter-cli/src/main/java/io/micronaut/starter/cli/command/ListFeatures.java (hunk 1 above)
Comment (timyates) [reply starter-cli/src/main/java/io/micronaut/starter/cli/command/ListFeatures.java:97]: Done!  PTAL https://github.com/micronaut-projects/micronaut-starter/pull/1063/commits/b189ca635115b2784f6b300e6a62b7f6f24064a7

**Review by sdelamo — APPROVED** [review]

**Pull request merged**